Dev/Checkman/checkman.10s.rb

```diff
@@ -49,9 +49,26 @@
   },
 }
 
+PRIORITY_ORDER = [ICON_OK, ICON_OK_CHANGING, ICON_FAIL, ICON_FAIL_CHANGING, ICON_UNDETERMINED, ICON_UNDETERMINED_CHANGING]
+
+# TODO: What is the parameter to prevent the line to be dimmed? This is
+# the only way I could figure out
+# NO_DIM=" bash=/bin/true terminal=false "
+NO_DIM=" bash=/bin/true terminal=false "
```

Comment:
This is totally an OSX thing.  Your workaround is fine.  If you don't want it dimmed it must be an action or submenu.  The other option would be to set the `color=black` but that's not friendly to darkmode...